example-crates/wallet_esplora_async/src/main.rs

```diff
@@ -56,12 +56,18 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let (update_graph, last_active_indices) = client
         .scan_txs_with_keychains(keychain_spks, None, None, STOP_GAP, PARALLEL_REQUESTS)
         .await?;
+    let update = Update {
+        graph: update_graph,
+        ..Default::default()
+    };
+    wallet.apply_update(update)?;
+
     let missing_heights = wallet.tx_graph().missing_heights(wallet.local_chain());
```

Comment:
```suggestion
    let missing_heights = update_graph.missing_heights(wallet.local_chain());
```

Reply:
Ouch of course, this is the right way to do it! I'm updating the code.